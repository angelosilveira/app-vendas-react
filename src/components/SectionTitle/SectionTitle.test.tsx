import { screen } from '@testing-library/react';
import { renderWithTheme } from 'src/utils/tests/renderWithTheme';

import { SectionTitle } from '.';

describe('<SectionTitle />', () => {
  it('should render the heading', () => {
    const { container } = renderWithTheme(
      <SectionTitle title="app de vendas" />
    );

    expect(
      screen.getByRole('heading', { name: /app de vendas/i })
    ).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
