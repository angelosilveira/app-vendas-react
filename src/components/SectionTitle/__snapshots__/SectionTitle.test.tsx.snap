// Vitest Snapshot v1

exports[`<SectionTitle /> > should render the heading 1`] = `
.c0 {
  border-bottom: 1px solid #e9e9e9;
  margin-bottom: 1rem;
  padding-bottom: 1rem;
  padding-top: 1rem;
}

.c0 h2 {
  font-size: 2.0rem;
  color: #030517;
  margin-bottom: 0;
}

<div
  class="c0"
>
  <h2>
    app de vendas
  </h2>
</div>
`;
